zero-thrs-control/src/thrs/cli/simulation_controls.py

```diff
@@ -49,7 +49,7 @@
 
 CONTROL_PARAMS = {"THRUSTERS": ThrustersParameters()}
 
-CONTROLS = {"THRUSTERS": ThrustersControl(CONTROL_PARAMS["THRUSTERS"])}
+CONTROLS = {"THRUSTERS": ThrustersControl(CONTROL_PARAMS["THRUSTERS"], datetime.now())} #TODO: should retreive time from somewhere else?
```

Comment:
```suggestion
CONTROLS = {"THRUSTERS": ThrustersControl(CONTROL_PARAMS["THRUSTERS"], datetime.now())} #TODO: should init control in simulation run instead of statically
```